src/hardware/mixer.cpp

```diff
@@ -267,17 +267,23 @@ static void set_global_crossfeed(mixer_channel_t channel)
 	// Global crossfeed
 	auto crossfeed = 0.0f;
 
-	const std::string crossfeed_pref = sect->Get_string("crossfeed");
-	if (crossfeed_pref == "on") {
-		constexpr auto default_crossfeed_strength = 0.40f;
-		crossfeed = default_crossfeed_strength;
-	} else if (crossfeed_pref == "off") {
-		crossfeed = 0.0f;
-	} else if (const auto p = parse_percentage(crossfeed_pref); p) {
+	const std::string_view crossfeed_pref = sect->Get_string("crossfeed");
+	const auto crossfeed_pref_has_bool = parse_bool_setting(crossfeed_pref);
+
+	if (crossfeed_pref_has_bool) {
+		if (*crossfeed_pref_has_bool == true) {
+			constexpr auto default_crossfeed_strength = 0.40f;
+			crossfeed = default_crossfeed_strength;
+		} else { // false
+			crossfeed = 0.0f;
+		}
+	}
```

Comment:
I think because a value of "1" is treated as "enabled", `crossfeed = 1` would set crossfeed to 40, actually. Then `crossfeed = 0` would work fine by fluke because that means disabled anyway 😄 

I have a proposed solution for this in a later comment below (basically excluding `0` and `0` from `parse_bool_setting`, and only handling them as extra special values when parsing "pure bool" config settings.